requirements/base.txt

```diff
@@ -21,8 +21,8 @@ pyopenssl==23.0.0
 # Needed for supporting Protocol in Python 3.7, Protocol class became public with python3.8
 typing_extensions~=4.4.0
 
-# NOTE: regex is not a direct dependency of SAM CLI, but pin to 2021.9.30 due to 2021.10.8 not working on M1 Mac - https://bitbucket.org/mrabarnett/mrab-regex/issues/399/missing-wheel-for-macosx-and-the-new-m1
-regex==2021.9.30
+# NOTE: regex is not a direct dependency of SAM CLI, exclude version 2021.10.8 due to not working on M1 Mac - https://github.com/mrabarnett/mrab-regex/issues/399
+regex!=2021.10.8
```

Comment:
in my opinion it is better to pin to a specific version, so the SAM CLI behaviour will be the same for everyone. Can you change this to pin to the latest stable version.

Reply:
good point